src/drawer.canvas.js

```diff
@@ -48,6 +48,25 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.Canvas, {
 
         this.clearWave();
     },
+    
+    updateWidthnHeight: function () {
```

Comment:
Yes, including an example would be nice as well (don't forget to include it at the bottom of index.html, the frontpage).